packages/nimble-components/src/wafer-map/tests/prerendering.coloring.spec.ts

```diff
@@ -50,9 +50,13 @@ describe('Wafermap Prerendering module', () => {
             });
 
             it('should have black fill style for all dies', () => {
-                for (const dieRenderInfo of prerenderingModule.diesRenderInfo) {
-                    expect(dieRenderInfo.fillStyle).toEqual(emptyDieColor);
-                }
+                const expectedValues = Array(18).fill(emptyDieColor);
```

Comment:
There are tests in the file that are disabled with xdescribe, I would expect that to be enabled and fixed to address the issue

Reply:
Sorry, missed those